src/evaluators/utility/PHAL_IsAvailable.hpp

```diff
@@ -0,0 +1,46 @@
+//*****************************************************************//
+//    Albany 3.0:  Copyright 2016 Sandia Corporation               //
+//    This Software is released under the BSD license detailed     //
+//    in the file "license.txt" in the top-level Albany directory  //
+//*****************************************************************//
+
+#ifndef PHAL_IS_PARAM_AVAILABLE_HPP
+#define PHAL_IS_PARAM_AVAILABLE_HPP
+
+namespace PHAL
+{
+
+enum class FieldScalarType : int {
```

Comment:
It doesn't look like you use this enum in this file. Maybe leftover from copy+paste of LandIce_ProblemUtils.hpp?